extensions/wikia/EditPageLayout/EditPageLayoutAjax.class.php

```diff
@@ -9,18 +9,6 @@ static private function resolveWikitext( $content, $mode, $page, $method, $secti
 		global $wgRequest, $wgTitle, $wgOut, $wgEnableSlowPagesBlacklistExt;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$wgEnableSlowPagesBlacklistExt" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)